libs/tup-components/src/accounts/ManageAccount.module.css

```diff
@@ -31,36 +30,37 @@
     font-style: normal;
 }
 
+.tap-feature,
+.tap-separator {
+  margin-top: 1em;
+}
+
 .tap-header {
-    padding-top: 10px;
     padding-bottom: 10px;
-    margin-bottom: 1em;
     border-bottom: 1px solid #707070;
 }
-.tap-description {
-    margin-block: 1em;
+.tap-header:is(h2) {
+    font-size: var(--global-font-size--small);
 }
```

Comment:
**Why?** Cuz we should use `<h2>` for this **heading**, but Designer doesn't want the large default size of `<h2>`.